app/sections/view-native-desktop/macro.test.js

```diff
@@ -82,4 +82,155 @@ describe('view-native-desktop', () => {
+          },
+        },
+      },
+    };
+
+    const dummyApp = createTestHarness(macroWrapper, context);
+    request(dummyApp)
+      .get('/')
+      .then((res) => {
+        const $ = cheerio.load(res.text);
+
+        const nativeDesktopSectionTable = $('[data-test-id="view-section-table-native-desktop"]');
+        const minimumMemoryRequirementQuestionRow = nativeDesktopSectionTable.find('[data-test-id="view-section-table-row-minimum-memory-requirement"]');
+        const minimumMemoryRequirementInnerComponent = minimumMemoryRequirementQuestionRow
+          .find('div[data-test-id="view-section-table-row-component"]')
+          .find('[data-test-id="view-question-data-text-minimum-memory-requirement"]');
+
+        expect(nativeDesktopSectionTable.length).toEqual(1);
+        expect(minimumMemoryRequirementQuestionRow.length).toEqual(1);
+        expect(minimumMemoryRequirementQuestionRow
+          .find('div[data-test-id="view-section-table-row-title"]').text().trim()).toEqual('Minimum memory requirement');
+        expect(minimumMemoryRequirementInnerComponent.length).toEqual(1);
+        expect(minimumMemoryRequirementInnerComponent.text().trim()).toEqual('4GB');
+
+        done();
+      });
+  });
+
+  it('should render the additional storage requirements answer', (done) => {
+    const context = {
+      params: {
+        section: {
+          sections: {
+            'native-desktop-memory-and-storage': {
+              answers: {
+                'storage-requirements-description': 'You will need at least 2.5GB of free space on each device the application is installed.',
+              },
+            },
+          },
+        },
+      },
+    };
+
+    const dummyApp = createTestHarness(macroWrapper, context);
+    request(dummyApp)
+      .get('/')
+      .then((res) => {
+        const $ = cheerio.load(res.text);
+
+        const nativeDesktopSectionTable = $('[data-test-id="view-section-table-native-desktop"]');
+        const storageRequirementsDescriptionQuestionRow = nativeDesktopSectionTable.find('[data-test-id="view-section-table-row-storage-requirements-description"]');
+        const storageRequirementsDescriptionInnerComponent = storageRequirementsDescriptionQuestionRow
+          .find('div[data-test-id="view-section-table-row-component"]')
+          .find('[data-test-id="view-question-data-text-storage-requirements-description"]');
+
+        expect(nativeDesktopSectionTable.length).toEqual(1);
+        expect(storageRequirementsDescriptionQuestionRow.length).toEqual(1);
+        expect(storageRequirementsDescriptionQuestionRow
+          .find('div[data-test-id="view-section-table-row-title"]').text().trim()).toEqual('Additional storage requirements');
+        expect(storageRequirementsDescriptionInnerComponent.length).toEqual(1);
+        expect(storageRequirementsDescriptionInnerComponent.text().trim()).toEqual('You will need at least 2.5GB of free space on each device the application is installed.');
```

Comment:
you could refer back to the context for your string checks e.g. ```expect(storageRequirementsDescriptionInnerComponent.text().trim()).toEqual(context.params.section.sections['native-desktop-memory-and-storage'].answers['storage-requirements-description']);```

Reply:
Yeah, I was thinking about that, it'll be something I'll consider changing after I've got all the native desktop components done as I'd have to change it in all the existing tests.